components/unit/file-explorer/UnitFileExplorer.vue

```diff
@@ -102,10 +107,12 @@
             :is="componentForType"
             v-bind="{ fileManager, filename }"
             v-if="supportedTypes.has(fileType)"
+            @loading="onLoading"
           />
           <UnitFileExplorerViewerUnknown
             v-else
             v-bind="{ fileManager, filename }"
+            @loading="onLoading"
```

Comment:
I guess we don't need to listen to the loading event on `UnitFileExplorerViewerUnknown`